app/controllers/participating_seller_tickets_controller.rb

```diff
@@ -1,13 +1,25 @@
 # frozen_string_literal: true
 
+include Pagy::Backend
```

Comment:
I don't think that you need this since it's in the gemfile

Reply:
This is how pagy recommends using functions--as I think they're not scoped but rather recommended to be included